### PR TITLE
Doc: Update readme specs

### PR DESCRIPTION
**Pull request by Uchihabeats**
This PR include issue #29
  
Updating the README.md:
PUIS to Suspects
PUMS to Probable

### CODE REVIEW

**Comment by JHNLWHD** [comment]
@Uchihabeats first make an issue, update your PR description and reference it to your description, and lastly amend/update your commit message to `doc: update readme specs".

**Comment by Uchihabeats** [comment]
[](url)

**Comment by Uchihabeats** [comment]
@JHNLWHD kindly check if this is correct.

**Comment by JHNLWHD** [comment]
@Uchihabeats update this commit message to `doc: update readme specs`
image

File: README.md [hunk 1]
```diff
@@ -19,7 +19,7 @@
 | ------------- |-------------| -----| --- | -- |
 | **GET**      | `/cases` | Gets all cases |  | ``` [ { "id": 1, "Case": "ZP01",... } ] ``` |
 | **GET**      | `/cases/:id` | Gets specific case  | *id* | ```{ "id": 1, "Case": "ZP01",... }```
-| **GET**      | `/figures` | Retrieves current counts of PUIs, PUMs, and confirmed cases in Zamboanga City.  | *id* | ```{ "pui": { "Value": 75, ... }, "pum": { "Value": 2381, ... }, "confirmed": { "value": 5, ... } }```
-| **GET**      | `/puis` | Gets all PUI count |  | ``` [ { "id": 1, "Value": 93, "DataUpdatedAt": "2020-04-06T00:00:00Z... }, ... ] ``` |
-| **GET**      | `/pums` | Gets all PUM count |  | ``` [ { "id": 1, "Value": 2381, "DataUpdatedAt": "2020-04-06T00:00:00Z... }, ... ] ``` |
+| **GET**      | `/figures` | Retrieves current counts of Suspects, Probables, and confirmed cases in Zamboanga City.  | *id* | ```{ "suspects": { "Value": 75, ... }, "probable": { "Value": 2381, ... }, "confirmed": { "value": 5, ... } }```
```
Comment (silemi5) [inline README.md:22]: The key should be `suspect` instead of `suspects` in the sample response of `/figures`.
image

**Pull request merged**